src/decode.h

```diff
@@ -546,6 +550,9 @@ typedef struct Packet_
      */
     struct PktPool_ *pool;
 
+    uint16_t alproto;
+    uint8_t cachedflags;
```

Comment:
Something like 'cached_flow_flags'? Although it's not a pure cache/copy of the flow flags of course.